applications/ParticleMechanicsApplication/tests/cpp_tests/test_mpm_stress_principal_invariants_utilities.cpp

```diff
@@ -275,23 +275,23 @@ namespace Testing
         auto positive_hydrostatic = CreateHydrostaticStressVector();
         MPMStressPrincipalInvariantsUtility::CalculateStressInvariants(positive_hydrostatic, p, q, lode_angle);
 
-        KRATOS_CHECK_LESS_EQUAL((3.0e6 -         p)/         p, 1e-6);
-        KRATOS_CHECK_LESS_EQUAL((0.0   -         q)/         q, 1e-6);
-        KRATOS_CHECK_LESS_EQUAL((0.0   -lode_angle)/lode_angle, 1e-6);
```

Comment:
@philbucher  I just changed the interface, because here we divided by zero. Therefore, the tests failed in #8049.

Reply:
Anyway, I can wait for @bodhinandach.

Reply:
well then I guess you can go ahead

not sure how responsive @bodhinandach is these days